packages/ui/src/views/pull-request/pull-request-list.tsx

```diff
@@ -30,61 +30,65 @@ export function PullRequestList({ pullRequests, open_prs, closed_prs }: PullRequ
     })
   }, [headerFilter])
```

Comment:
Looks like you'll need `pullRequests` here.
```suggestion
  }, [headerFilter, pullRequests])
```

Reply:
done